orc8r/cloud/go/services/state/indexer/indexer.go

```diff
@@ -68,10 +69,11 @@ type Version uint32
 // NewIndexerVersion returns a new indexer version, ensuring it fits into
 // the required integer size.
 func NewIndexerVersion(version int64) (Version, error) {
-	capped := Version(version)
-	if int64(capped) != version {
-		return 0, fmt.Errorf("indexer version %v too large for %T", version, Version(0))
+	if version < 0 || version > math.MaxUint32 {
+		return 0, fmt.Errorf("version value %d is outside the boundaries of uint32 type", version)
 	}
+	capped := Version(version)
```

Comment:
Looks like the code scan is still flagging this -- any thoughts on how to appease the scanner?


<img width="844" alt="Screen Shot 2021-12-13 at 4 44 45 PM" src="https://user-images.githubusercontent.com/8029544/145900959-348b0c14-5346-4522-bd3a-0513b27be2d1.png">